database/localdb/localdb.go

```diff
@@ -0,0 +1,65 @@
+package localdb
+
+import (
+	"sync"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/tecbot/gorocksdb"
+
+	"chain/errors"
+)
+
+// DB provides access to a kv store.
```

Comment:
I think it is safe to call methods on a *DB concurrently, right? We should document that, since it is an exception to the usual convention (that it is not safe to call methods concurrently on most objects).